src/adapter/src/session/vars.rs

```diff
@@ -379,6 +379,17 @@ const PERSIST_COMPACTION_MINIMUM_TIMEOUT: ServerVar<Duration> = ServerVar {
     safe: true,
 };
 
+/// Controls the connection timeout to Cockroach.
+///
+/// Used by persist as [`mz_persist_client::cfg::DynamicConfig::consensus_connect_timeout`].
+const CRDB_CONNECT_TIMEOUT: ServerVar<Duration> = ServerVar {
```

Comment:
Missing from `SystemVars::iter`... intentional?

Reply:
Nooope missed that one 🙈